apps/content_types/content_types.py

```diff
@@ -49,6 +49,11 @@ class ContentTypesResource(superdesk.Resource):
             'type': 'string',
             'iunique': True,
         },
+        'item_type': {
+            'type': 'string',
+            'nullable': True,
```

Comment:
Why nullable is allowed? 
We can make migration and set `text` for already created entries + use `unique` instead of `content_type_single_item_type`

Reply:
you can have multiple `text` types (or `null` in this case), so that unique check won't work. I try to avoid migrations in general, users might not run them and here supporting both is not complex